### PR TITLE
fix: Unhandled scan status "PENDING"

### PR DESCRIPTION
**Pull request by s-hashimo**
The following error often occurs in our CI environment:

> Error: Unhandled scan status "PENDING". API response: {"registryId":"222222222222","repositoryName":"dummy","imageId":{"imageDigest":"sha256:08eec9208c975baa1df884a1009b38426b1a44e90048d7ae82f59ba268767c36","imageTag":"08eec9208c975baa1df884a1009b38426b1a44e90048d7ae82f59ba268767c36"},"imageScanStatus":{"status":"PENDING","description":"Pending start of the image scan."},"imageScanFindings":{"findings":[],"enhancedFindings":[]}}

And here is the AWS documentation:

https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_ImageScanStatus.html
> Valid Values: `IN_PROGRESS` | `COMPLETE | FAILED` | `UNSUPPORTED_IMAGE` | `ACTIVE` | `PENDING` | `SCAN_ELIGIBILITY_EXPIRED` | `FINDINGS_UNAVAILABLE`

### CODE REVIEW

**Review by iwasa-kosui — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -219,11 +219,11 @@ const main = async () => {
       },
       repositoryName: repository
     }).promise()
-    status = 'IN_PROGRESS'
+    status = 'PENDING'
```
Comment (alexjurkiewicz) [inline index.js:222]: Is this change necessary?

Comment (s-hashimo) [reply index.js:222]: I think the flow will be `PENDING` -> `IN_PROGRESS` -> `ACTIVE` / `COMPLETE`, but I'm not sure. 🙈 

The behavior should be the same either way, so I will revert the change if you want. 👍

**Review by alexjurkiewicz — APPROVED** [review]

**Comment by alexjurkiewicz** [comment]
LGTM thanks for the patch!
